src/nvim/main.c

```diff
@@ -1398,6 +1398,17 @@ static void load_plugins(void)
 }
 
 /*
+ * Read all the early plugin files (those are sourced before the vimrc)
+ */
+static void load_early_plugins(void)
```

Comment:
Didn't want to parametrize `load_plugins()`, but it could be implemented that way too:

~~~ c
load_plugins("early");
...
load_plugins("plugin");
~~~